### PR TITLE
Use default config for tests

### PR DESCRIPTION
**Pull request by whalemer**
A few tests rely on hardcoded groups, which breaks the test if they're different.

### CODE REVIEW

File: test/main.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ before('initialization', function () {
 	}
 	require('./../lib/process-manager').disabled = true;
 
+	config = Object.assign(config, require('../config/config-example'));
```
Comment (Zarel) [inline test/main.js]: Is the original loading still necessary?

Comment (whalemer) [reply test/main.js]: Is this modifying the `require` cache for config.js? If it is, it's still necessary so that you edit the cache for the right file.

Comment (Zarel) [reply test/main.js]: Oh, I think it might be.

Comment (Zarel) [reply test/main.js]:
```suggestion
	Object.assign(config, require('../config/config-example'));
```

In which case, we should probably stop writing it in a way that suggests it doesn't have side effects.

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**